src/status_im2/core.cljs

```diff
@@ -38,9 +37,8 @@
   (global-error/register-handler)
   (notifications/listen-notifications)
   (.addEventListener rn/app-state "change" #(re-frame/dispatch [:app-state-change %]))
-  (react-native-languages/add-change-listener #(fn [lang]
-                                                 (i18n/set-language lang)
-                                                 (i18n-resources/load-language lang)))
+  (i18n/set-language "en")
```

Comment:
There are two main reasons for forcing english language as of now:
- For our alpha release, we were requested to support only english. But, why to do it in this PR?
- We are animating two overlapping texts "Sign In" and "Sign In by syncing", so it is important that those strings are not translated, otherwise animation would break

Reply:
hmm, I thought we had already handled this. @alwx had a separate pr I'm pretty sure. If it's needed then it's fine to keep 👍